src/OrleansSQLUtils/CreateOrleansTables_PostgreSql.sql

```diff
@@ -0,0 +1,937 @@
+
+INSERT INTO OrleansQuery(QueryKey, QueryText)
+VALUES
+(
+	'ReadReminderRowsKey','
+	SELECT
+		GrainId,
+		ReminderName,
+		StartTime,
+		Period,
+		Version
+	FROM OrleansRemindersTable
+	WHERE
+		ServiceId = @ServiceId AND @ServiceId IS NOT NULL
+		AND GrainId = @GrainId AND @GrainId IS NOT NULL;
+');
+
+INSERT INTO OrleansQuery(QueryKey, QueryText)
+VALUES
+(
+	'ReadReminderRowKey','
+	SELECT
+		GrainId,
+		ReminderName,
+		StartTime,
+		Period,
+		Version
+	FROM OrleansRemindersTable
+	WHERE
+		ServiceId = @ServiceId AND @ServiceId IS NOT NULL
+		AND GrainId = @GrainId AND @GrainId IS NOT NULL
+		AND ReminderName = @ReminderName AND @ReminderName IS NOT NULL;
+');
+
+INSERT INTO OrleansQuery(QueryKey, QueryText)
+VALUES
+(
+	'ReadRangeRows1Key','
+	SELECT
+		GrainId,
+		ReminderName,
+		StartTime,
+		Period,
+		Version
+	FROM OrleansRemindersTable
+	WHERE
+		ServiceId = @ServiceId AND @ServiceId IS NOT NULL
+		AND GrainHash > @BeginHash AND @BeginHash IS NOT NULL
+		AND GrainHash <= @EndHash AND @EndHash IS NOT NULL;
+');
+
+INSERT INTO OrleansQuery(QueryKey, QueryText)
+VALUES
+(
+	'ReadRangeRows2Key','
+	SELECT
+		GrainId,
+		ReminderName,
+		StartTime,
+		Period,
+		Version
+	FROM OrleansRemindersTable
+	WHERE
+		ServiceId = @ServiceId AND @ServiceId IS NOT NULL
+		AND ((GrainHash > @BeginHash AND @BeginHash IS NOT NULL)
+		OR (GrainHash <= @EndHash AND @EndHash IS NOT NULL));
+');
+
+INSERT INTO OrleansQuery(QueryKey, QueryText)
+VALUES
+(
+	'InsertOrleansStatisticsKey','
+
+	START TRANSACTION;
+	INSERT INTO OrleansStatisticsTable
+	(
+		deploymentid,
+		id,
+		hostname,
+		name,
+		isvaluedelta,
+		statvalue,
+		statistic
+	)
+	SELECT
+		@DeploymentId,
+		@Id,
+		@HostName,
+		@Name,
+		@IsValueDelta,
+		@StatValue,
+		@Statistic;
+	COMMIT;
+');
+
+
+
+CREATE FUNCTION delete_reminder_row(
+    ServiceIdArg    OrleansRemindersTable.ServiceId%TYPE, 
+    GrainIdArg      OrleansRemindersTable.GrainId%TYPE, 
+    ReminderNameArg OrleansRemindersTable.ReminderName%TYPE, 
+    VersionArg      OrleansRemindersTable.Version%TYPE
+)
+  RETURNS TABLE(row_count integer) AS
+$func$
+DECLARE
+    RowCountVar int := 0;
+BEGIN
+  
+  
+    DELETE FROM OrleansRemindersTable
+	WHERE
+		ServiceId = ServiceIdArg AND ServiceIdArg IS NOT NULL
+		AND GrainId = GrainIdArg AND GrainIdArg IS NOT NULL
+		AND ReminderName = ReminderNameArg AND ReminderNameArg IS NOT NULL
+		AND Version = VersionArg AND VersionArg IS NOT NULL;
+        
+    GET DIAGNOSTICS RowCountVar = ROW_COUNT;
+
+    RETURN QUERY SELECT RowCountVar;
+    
+END
+$func$ LANGUAGE plpgsql;
+
+INSERT INTO OrleansQuery(QueryKey, QueryText)
+VALUES
+(
+	'DeleteReminderRowKey','
+	SELECT * FROM delete_reminder_row(
+        @ServiceId,
+        @GrainId,
+        @ReminderName,
+        @Version
+    );
+');
+
+INSERT INTO OrleansQuery(QueryKey, QueryText)
+VALUES
+(
+	'DeleteReminderRowsKey','
+	DELETE FROM OrleansRemindersTable
+	WHERE
+		ServiceId = @ServiceId AND @ServiceId IS NOT NULL;
+');
+
```

Comment:
empty lines removed at the end of file (WHERE clause not changed).  I think that's what you mean.